sdcm/wait.py

```diff
@@ -156,12 +156,14 @@ def wait_for_log_lines(node, start_line_patterns, end_line_patterns, start_timeo
         started = any(start_follower)
         while not started and (time.time() - start_time < start_timeout):
             started = any(start_follower)
+            time.sleep(0.05)
```

Comment:
Is it intentional that we have 2 different values for 2 sleeps?

Reply:
no, forgotten about this one.

Reply:
fixed